caliban/docker/push.py

```diff
@@ -18,10 +18,15 @@
 
 """
 
+import json
 import subprocess
 
+from absl import logging
 
-def _image_tag_for_project(project_id: str, image_id: str) -> str:
+
```

Comment:
would having a `tag` parameter here that defaults to `latest` be more flexible than just `include_tag`? We don't really tag things differently now, but perhaps in the future.